sources/install/package_web.sh

```diff
@@ -894,6 +894,20 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    zap_version=$(curl -s "https://www.zaproxy.org/download/" | grep -E -o "ZAP [0-9]+(\.[0-9]+)+" | head -1 | cut -d' ' -f2)
+    wget "https://github.com/zaproxy/zaproxy/releases/download/v${zap_version}/ZAP_${zap_version}_Linux.tar.gz" -O /tmp/zap.tar.gz
+    tar -xf /tmp/zap.tar.gz -C /tmp/
+    mkdir /opt/tools/ZAP
+    mv /tmp/ZAP_${zap_version}/* /opt/tools/ZAP/
```

Comment:
Finally, we're going to do something a little cleaner. Please, remove the **mkdir** and replace the following command with :
**mv /tmp/ZAP_${zap_version}/ /opt/tools/ZAP**

Reply:
ok...
so I reverted to what I did before.